test/integration_partners/listing_statuses_test.rb

```diff
@@ -0,0 +1,31 @@
+require "test_helper"
+
+module IntegrationPartners
+  class ListingStatusesTest < EasyBrokerTestBase
+    attr_reader :statuses
+
+    def setup
+      @statuses = EasyBroker::IntegrationPartners::ListingStatuses.new(EasyBroker::ApiClient.new)
+    end
+
+    def test_search
+      stub_verb_request(:get, '/integration_partners/listing_statuses', query: { page: 1 }).
```

Comment:
This is probably a bad example because we shouldn't pass the page in the query hash in the future. You can leave it but update it in the other PR.

Reply:
In this case `query` is used to stub the request using WebMock and is not related to the `query` param that we used to handle in the `send_request` method.